lms/djangoapps/courseware/access.py

```diff
@@ -20,7 +20,7 @@
 
 from opaque_keys.edx.keys import CourseKey, UsageKey
 
-from util import milestones_helpers as milestones_helpers
+from util import milestones_helpers
```

Comment:
This is imported again below, with specific functions that are needed, on line 51.